src/Tes.Runner.Test/Docker/DockerExecutorTests.cs

```diff
@@ -54,7 +54,7 @@ public async Task RunOnContainerAsync_DockerClientReturnsAuthNeeded_CallsContain
             Models.RuntimeOptions runtimeOptions = new();
             try
             {
-                var result = await executor.RunOnContainerAsync(new("msftsc022830.azurecr.io/broadinstitute/gatk", "4.5.0.0-squash", [""], default, default, runtimeOptions));
+                var result = await executor.RunOnContainerAsync(new("msftsc022830.azurecr.io/broadinstitute/gatk", "4.5.0.0-squash", [""], default, default, runtimeOptions, default));
```

Comment:
should we use a public image instead?

Reply:
This is a unit test. No call to the docker engine is actually made.

Having said that, placeholder image values would be appropriate here (e.g. "executor/image:tag")